modules/account-lib/src/coin/baseCoin/iface.ts

```diff
@@ -130,3 +130,24 @@ export type FeeOptions = {
   formula?: 'fixed' | 'feeRate' | 'perKB' | 'custom';
   type?: 'base' | 'max' | 'tip';
 };
+
+/**
+ * The validity window signals when a transaction can be included in the network.
+ *  ValidityWindow information is stored as part of the transaction data in the indexer and wallet platform, unless it is absolutely necessary.
+ */
+export interface ValidityWindow {
+  firstValid?: number;
+  lastValid?: number;
+  minDuration?: number;
+  maxDuration?: number;
+  unit?: 'block' | 'seconds' | 'milliseconds';
```

Comment:
this isn't enough to cover all validity window scenarios. from the doc, its possible to specify a window `in seconds after certain blocks`.

this might look something like:
```
{
  firstValid: 1234, // block number
  maxDuration: 120 // 120s after the block number,
  unit: 'block' // problem here since fields are in different units.
}
```

i think we should create a new type called Quantity:
```
export interface Quantity {
   value: number,
   unit: 'block' | 'seconds' | 'milliseconds'
}
```

Reply:
CC @shiksb @argjv

Reply:
Until we not have defined this interfaces the pr will be rejected.